c_binding/libsg.c

```diff
@@ -297,6 +297,13 @@ int _sg_io_vpd(char *err_msg, int fd, uint8_t page_code, uint8_t *data)
                                  sense_err_msg);
             }
         }
+        /* NVMe disk does not support SCSI VPD page 0x00 which fallback to here
+         */
+        if (ioctl_errno == ENOTTY) {
```

Comment:
Is this the only error code that an NVMe device can kick back when it doesn't support 0x00?